src/SFA-DAS-HMRC-API-Stub/Commands/GetFractionCalcDateCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using SFA.DAS.HMRC.API.Stub.Data.Repositories;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.HMRC.API.Stub.Commands
+{
+    public class GetFractionCalcDateCommand : ICommand<GetFractionCalcDateRequest, GetFractionCalcDateResponse>
+    {
+        private readonly IFractionsRepository _fractionsRepository;
+
+        public GetFractionCalcDateCommand(IFractionsRepository fractionsRepository)
+        {
+            _fractionsRepository = fractionsRepository ?? throw new ArgumentException("fractionsRepository cannot be null");
+        }
+
+        public async Task<GetFractionCalcDateResponse> Get(GetFractionCalcDateRequest request)
+        {
+            var result = await _fractionsRepository.GetByLastCalcDate();

```

Comment:
Could be GetLastCalcDate as the method doesn't get it by the last calculated date